skimage/measure/tests/test_regionprops.py

```diff
@@ -500,3 +500,16 @@ def foo():
 def test_incorrect_coordinate_convention():
     with testing.raises(ValueError):
         regionprops_default(SAMPLE, coordinates='xyz')[0]
+
+
+def test_to_dict():
+    regions = regionprops(SAMPLE)
+    out = to_dict(regions)
+    assert 'label' in out
+    assert 'bbox-0' in out
+
+    regions = regionprops(SAMPLE)
+    out = to_dict(regions, ['area'], separator='+')
+    assert 'label' in out
+    assert 'bbox+1' in out
+    assert 'area' in out
```

Comment:
These aren't a bad start! But you should check correct dtypes for many columns, and you should check that everything works as expected if you pass in *all* regionprops as desired columns: the shape of the table, the number of columns, etc.